src/SigSpec.Json.Middleware/SigSpecUIBuilderExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SigSpec.Core;
+using System;
+using System.Collections.Generic;
+
+namespace SigSpec.Middleware
```

Comment:
Please use Microsoft.AspNetCore.Builder namespace here
same as in NSwag here: 
https://github.com/RicoSuter/NSwag/blob/master/src/NSwag.AspNetCore/Extensions/NSwagApplicationBuilderExtensions.cs#L23
(also use a aligned class name to avoid name clashes)

Reply:
Changed the namespace